core/unit_test/CMakeLists.txt

```diff
@@ -1284,6 +1284,29 @@ KOKKOS_ADD_EXECUTABLE_AND_TEST(
   ARGS "one 2 THREE"
 )
 
+KOKKOS_ADD_EXECUTABLE(
+  CoreUnitTest_CMakeTriBITSCompatibility
+  SOURCES UnitTest_CMakeTriBITSCompatibility.cpp
+)
+
+SET(Kokkos_CoreUnitTest_CMakeTriBITSCompatibilityDisable_DISABLE ON)
```

Comment:
To check my understanding, the confirmation that this test passes (i.e. does not run thanks to the _DISABLE property) is that the unit test will not terminate with `EXIT_FAILURE`, even though the test lacks the required  `KOKKOS_TEST_TRIBITS_COMPATIBILITY` or `--kokkos-test-tribits-compatibility` matches required for success, does that sound correct?

Reply:
I'm referring to L1292-1296, my highlight block didn't carry over when I commented

Reply:
Correct, if we actually tried to run that command it would fail.

Reply:
I added a test with the WILL_FAIL property to ensure that it would indeed fail

Reply:
```
$ ctest --test-dir cmake-build-default -C Debug -R TriBITS

Internal ctest changing into directory: /Users/qdi/CLionProjects/kokkos/cmake-build-default
Test project /Users/qdi/CLionProjects/kokkos/cmake-build-default
    Start 19: Kokkos_CoreUnitTest_CMakeTriBITSCompatibilityWillFail
1/3 Test #19: Kokkos_CoreUnitTest_CMakeTriBITSCompatibilityWillFail ......   Passed    0.62 sec
    Start 20: Kokkos_CoreUnitTest_CMakeTriBITSCompatibilityExtraArgs
2/3 Test #20: Kokkos_CoreUnitTest_CMakeTriBITSCompatibilityExtraArgs .....   Passed    0.02 sec
    Start 21: Kokkos_CoreUnitTest_CMakeTriBITSCompatibilityEnvironment
3/3 Test #21: Kokkos_CoreUnitTest_CMakeTriBITSCompatibilityEnvironment ...   Passed    0.01 sec

100% tests passed, 0 tests failed out of 3

Label Time Summary:
Kokkos    =   0.64 sec*proc (3 tests)

Total Test time (real) =   0.66 sec